examples/visualize_graph.py

```diff
@@ -0,0 +1,9 @@
+import torch
```

Comment:
Consider deleting this file or adding back the ability to visualize the graph. Maybe start with outputting the layers to the console.